problems/euler/number2/java/kendellprather/FibTest.java

```diff
@@ -0,0 +1,14 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+import Fib.java;
+
+public class FibTest{
+
+	@Test
+	public void testRun() {
+		result = Fib.run();
+		expected = 4613732
+		assertEquals(expected, result)
```

Comment:
I think this is close, but the return value of Fib.run() is void.  Also I don't think this will even compile.  Please confirm and clean up.